docs/quickstart.md

```diff
@@ -50,10 +50,10 @@ You can access the JupyterLab instance by using the URL's that are printed out b
 
 Please see the [dnsrecords documentation](./docs/dnsrecords.md) guide for more information on how to access the JupyterLab instance.
 
-## Cancelling all Jobs
+## Cancelling a Job
 
-**NOTE: This will kill all jobs running on queue `default` and job-set-id `testing`**
+To cancel a job you can use the following command:
 
 ```
-python3 ./docs/dev/cancel.py
+python3 -m armada_jupyter cancel {URL}
```

Comment:
For docs and readme, I would like to see what this URL should be?

Reply:
Yeah. I definitely need to document this better. Thanks